docs/en/latest/pubsub/kafka.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Apache Kafka
+keywords:
+  - APISIX
+  - Pub-Sub
```

Comment:
Could we use PubSub like pubsub in other places?

Reply:
All `Pub-Sub` replaced to `PubSub`